obspy/taup/tau_branch.py

```diff
@@ -489,9 +487,9 @@ def _robust_resize(arr, new_size):
         try:
             arr.resize(new_size)
         except ValueError:
-            msg = ('Resizing a TauP array inplace failed due to the existence'
-                   ' of other references to the array, creating a new array. '
-                   'See Obspy #2280.')
-            warnings.warn(msg)
+            # msg = ('Resizing a TauP array inplace failed due to the '
+            #        'existence of other references to the array, creating '
+            #        'a new array. See Obspy #2280.')
+            # warnings.warn(msg)
```

Comment:
Hmm.. I would assume this warning was put in there for a reason. Maybe leave it in and just catch it where it is raised in tests?

https://github.com/obspy/obspy/commit/223f0b4c7526070e1392dc8e8ca7d123a33e526e

Reply:
Yeah, that could be done, but I have the impression it is raised very often under usual circumstances. And I do not feel that there is a big disadvantage using np.resize which satisfies this warning.
@d-chambers What is your opinion on the matter? I think you added the warning.

With silencing I really meant removing. If it should just be silenced: There are a lot of other warnings during the tests, I guess these could then be fixed in a large effort to get rid of all warnings in one rush (after 1.2).

Reply:
> I guess these could then be fixed in a large effort to get rid of all warnings in one rush (after 1.2).

Yes, after 1.2.0 please. I know there are a lot of warnings, usually just nobody takes them on. We might also want a mechanism to make sure no warnings are introduced by PRs, because we clean them up every now and then but I think many PRs bring in new ones. (e.g. catch all warnings in test runner and raise a test fail if warnings appeared.. might be to rough though, because it might be hard to get of all warnings on all test runs across all platforms etc)

Reply:
> Yes, after 1.2.0 please.

I agree.


After rereading #2280, I think you are right and the warning has a reason to be there. It is more of a reminder that there is a minor bug somewhere in taupy. But I think the user should not be bothered with it. I suggest we merge this PR as is and I am going to open a new issue tracking this bug.